python/k8s.py

```diff
@@ -86,13 +91,18 @@ def k3d(force):
         LOGGER.info("Could not find k3d")
     if which("k3d"):
         found_k3d_version = re.match('k3d version (.+)', check_output(['k3d', '--version'])).group(1)
-    if not force and found_k3d_version != k3d_version:
+    if not force and version.parse(found_k3d_version) < version.parse(k3d_version):
```

Comment:
I'm not sure we want to allow greater version than the required one.
@Konubinix what do you think?

Reply:
@glehmann @Konubinix What about limiting upper version to any version strictly below next upper major version ? Something like if 2.13 is required, then everything between 2.13 and < 3.0 is fine.